pywinauto/base_wrapper.py

```diff
@@ -826,4 +837,94 @@ def set_focus(self):
         """Set the focus to this element"""
         pass
 
+    # -----------------------------------------------------------
+    def wait_visible(self, timeout, retry_interval):
+        """
+        Waiting until control is visible
```

Comment:
-> `Wait until...`
Also please document parameters similar to other methods. It would be good to take these params description from old doc strings as they have proven good enough English. ;)